tracer/src/Datadog.Trace/RemoteConfigurationManagement/Protocol/RcmConfigState.cs

```diff
@@ -22,7 +22,7 @@ public RcmConfigState(string id, int version, string product, uint applyState, s
         public string Id { get; }
 
         [JsonProperty("version")]
-        public int Version { get; }
+        public long Version { get; }
```

Comment:
we have a `uint` in `ApplyState` below, should that be a `ulong`?

Reply:
good point, fixed in e2e493699